.gitmodules

```diff
@@ -1,19 +1,3 @@
-[submodule "third_party/safe-contracts"]
-	path = third_party/safe-contracts
-	url = https://github.com/gnosis/safe-contracts
-	ignore = dirty
-[submodule "third_party/v2-periphery"]
-	path = third_party/v2-periphery
-	url = https://github.com/Uniswap/v2-periphery
-	ignore = dirty
-[submodule "third_party/v2-core"]
-	path = third_party/v2-core
-	url = https://github.com/Uniswap/v2-core
-	ignore = dirty
-[submodule "third_party/openzeppelin-contracts"]
-	path = third_party/openzeppelin-contracts
-	url = https://github.com/OpenZeppelin/openzeppelin-contracts
-	ignore = dirty
 [submodule "third_party/contracts-elcol/third_party/canonical-weth"]
```

Comment:
Are you replacing the remaining ones too?

Reply:
This is the only place we version them. Thats why I suggested we make a repo for them. They are not used in the code.

Reply:
Ok - yes then create a fork in a separate repo and build the image there, like we do for the others :)